lychee.toml

```diff
@@ -0,0 +1,60 @@
+################################################################################
+# Config for the link checker lychee.
+#
+# Download & learn more at:
+# https://github.com/lycheeverse/lychee
+#
+# Example config:
+# https://github.com/lycheeverse/lychee/blob/master/lychee.example.toml
+#
+# Run `lychee . --dump` to list all found links that are being checked.
+################################################################################
+
+# Maximum number of concurrent link checks.
+# Workaround for "too many open files" error on MacOS, see https://github.com/lycheeverse/lychee/issues/1248
+max_concurrency = 32
+
+
+# URLs to check (supports regex). Has preference over all excludes.
+exclude_path = [
+  # Unfortunately lychee doesn't yet read .gitignore https://github.com/lycheeverse/lychee/issues/1331
+  # The following entries are there because of that:
+  "__pycache__",
+  "_deps/",
+  ".pixi",
+  "build",
+  "docs/python/",
+  "rerun_cpp/docs/html",
+  "rerun_cpp/docs/xml",
+  "rerun_py/site/",
+  "target_ra",
+  "target_wasm",
+  "target",
+  "rerun_cpp/tmp/",
+
+  # Actually ignored files beyond .gitignore
+  "scripts/screenshot_compare/assets/templates/",
+]
+
+# Exclude URLs and mail addresses from checking (supports regex).
+exclude = [
+  '.*?speculative-link',                                      # Skip speculative links
+  '.github/workflows/.*.yml',                                 # GitHub action workflows cause issues on CI.
+  '\$relpath\^',                                              # Relative paths as used by rerun_cpp's doc header.
+  '%7B%7B.+%7D%7D',                                           # Ignore things that look like links because of `href` but are actually replacement strings. The URL escaping is for '{{.*}}'
```

Comment:
This seems to be escaping for `{{.+}}` instead:

```js
> decodeURIComponent("%7B%7B.+%7D%7D") === '{{.+}}'
< true
```

Reply:
ah typo in the comment. yes that was intentional :)